spacewalk/certs-tools/spacewalk-certs-tools.changes

```diff
@@ -1,3 +1,5 @@
+- Use RES bootstrap repository as failback repo when bootstrapping CentOS (bsc#1173556)
```

Comment:
`failback` does not exist. I think you mean `fallback`.
This exists also in some comments in the code.

Reply:
Thanks for the comment! I'll fix this and double check the code comments as well